tools/fetch-external.sh

```diff
@@ -49,32 +34,17 @@ cd shared/meta-balena/ && {
 } &
 
 # get integrations
-cd pages/learn/develop/integrations/ && {
-  curl -O -L https://raw.githubusercontent.com/balenalabs/google-iot/master/README.md
-  mv README.md google-iot.md
-  curl -O -L https://raw.githubusercontent.com/balena-io-playground/balena-azure-iot-hub/master/README.md
-  mv README.md azure-iot-hub.md
-  cd -
-} &
+curl -o pages/learn/develop/integrations/google-iot.md -L https://raw.githubusercontent.com/balenalabs/google-iot/master/README.md &
+curl -o pages/learn/develop/integrations/azure-iot-hub.md -L https://raw.githubusercontent.com/balena-io-playground/balena-azure-iot-hub/master/README.md &
 
 # Masterclasses
-cd pages/learn/more/masterclasses/ && {
-  curl -O -L https://raw.githubusercontent.com/balena-io/balena-cli-masterclass/master/README.md
-  mv README.md cli-masterclass.md
-  curl -O -L https://raw.githubusercontent.com/balena-io/balena-cli-advanced-masterclass/master/README.md
-  mv README.md advanced-cli.md
-  curl -O -L https://raw.githubusercontent.com/balena-io/balenaos-masterclass/master/README.md
-  mv README.md host-os-masterclass.md
-  curl -O -L https://raw.githubusercontent.com/balena-io/services-masterclass/master/README.md
-  mv README.md services-masterclass.md
-  curl -O -L https://raw.githubusercontent.com/balena-io-projects/balena-fleet-management-masterclass/master/README.md
-  mv README.md fleet-management.md
-  curl -O -L https://raw.githubusercontent.com/balena-io/debugging-masterclass/master/README.md
-  mv README.md device-debugging.md
-  curl -O -L https://raw.githubusercontent.com/balena-io/docker-masterclass/master/README.md
-  mv README.md docker-masterclass.md
-  cd -
-} &
+curl -o pages/learn/more/masterclasses/cli-masterclass.md -L https://raw.githubusercontent.com/balena-io/balena-cli-masterclass/master/README.md &
```

Comment:
I went with the explicit full path here rather than the `cd` because I think it's clearer this way (you can see for each request precisely where it'll go), it'll be easy to change the destinations for individual files as necessary, and it's easier for me to parallelize (I'm sure it could be done with the subshell format but I'm not 100%, and when combined with the other points I decided it wasn't worth working it out)